classexercises-lesson18

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<title>Exercises lesson 18</title>
+	<script>
+		/* 1.) Write a JavaScript program to calculate the factorial of a number. In mathematics, the factorial of a non-negative integer n, denoted by n!, is the product of all positive integers less than or equal to n. For example, 5! = 5 x 4 x 3 x 2 x 1 = 120*/
+		function factoriel(number) {
+			if (number === 0)
+			 {
+			 	return 1;
+				
+			} 
+				return number * factoriel(number-1)
+				
+			}
+			console.log(factoriel(5));
+
+		/* 2.) Write a JavaScript program to find the greatest common divisor (gcd) of two positive numbers.*/
+
+		function gcd(number1, number2) {
+			if ( number2 <= 0) {
+				return number1;
+			}
+			return gcd(number2, number1 % number2);
+		};
+		console.log(gcd(100, 10));
+
+		/* 3.) Write a JavaScript program to get the first n Fibonacci numbers. Note : The Fibonacci Sequence is the series of numbers: 0, 1, 1, 2, 3, 5, 8, 13, 21, 34, . . . Each subsequent number is the sum of the previous two.*/
+
+		
+
+        /* 4.) Write a JavaScript program to check whether a number is even or not.*/
+
+		function number(n) {
```

Comment:
Nice try, can you use recursion to solve it

Reply:
ok sir i will try and share